guide/xml/portfile-keywords.xml

```diff
@@ -277,5 +277,24 @@
       </listitem>
     </varlistentry>
 
+    <varlistentry>
+      <term>use_xcode</term>
+
+      <listitem>
+        <para>Available in 2.6.0 and later. By default, it is assumed on
+        <code>macosx</code>that ports will not need tools from Xcode.app
```

Comment:
Shouldn't there be a space after `</code>` here?

Reply:
Fixed